docs(workers): Add self-managed workers to website (#2180)
hashicorp/boundary: website/content/docs/oss/configuration/worker/kms-worker.mdx

## KMS Worker Configuration

> KMS Workers require a `name` field. This specifies a unique name of this worker within the Boundary cluster. This value
> can be a direct name string, can refer to a file on disk (file://) from which an name will be read; or an env
> var (env://) from which the name will be read.

Reviewer: this might be a little cleaner as a list, something like:

```suggestion
KMS Workers require a `name` field. This specifies a unique name of this worker within the Boundary cluster. 

The `name` value can be:
- a direct name string
- a reference to a file on disk (`file://`) from which the name is read
- an env var (`env://`) from which the name is read.
```

Author: Thanks- I've updated this!
